tests/lib.rs

```diff
@@ -39,20 +39,22 @@ fn test_queries_from_file(f: &Path, name: &str) -> Result<i32, i32> {
     f.read_to_string(&mut s).unwrap();
     let lines: Vec<String> = s
         .lines()
-        .filter(|l| !l.is_empty() && !l.starts_with("#"))
+        .filter(|l| !l.is_empty() && !l.starts_with("#") && !l.starts_with("--") && !l.starts_with("/*"))
```

Comment:
Oh, good point. I'll fix that soon and re-push. OK to hit merge after that?

Reply:
Yes, absolutely!

Reply:
Also, parse_file() in the same file seems like it would have the same problem.

Reply:
Yes, you're totally right -- let's remove it from `parse_file()` too.

Reply:
Unfortunately but unsurprisingly, that makes it fail the parse_comments test. Should I remove tests with /**/ or just let it be as-is? (I worry it'll cause problems elsewhere in the codebase too if something relies on it handling those.)

Reply:
Hmm, how about we change it to `!(l.starts_with("/*") && l.ends_with("*/"))`? That covers the case in that file, and any single-line case using that comment syntax.

In the long term, we'll perhaps want to move comment parsing into the main parser logic rather than input preprocessing and handle all these cases properly.

Reply:
Sounds great, thanks! I'm going to do that for both functions so they're at least somewhat more consistent, though I agree this is probably just a short-term approach.